dataland-frontend/src/components/forms/parts/elements/basic/UploadDocumentsForm.vue

```diff
@@ -8,6 +8,7 @@
       :maxFileSize="DOCUMENT_UPLOAD_MAX_FILE_SIZE_IN_BYTES"
       invalidFileSizeMessage="{0}: Invalid file size, file size should be smaller than {1}."
       :auto="false"
+      :multiple="true"
```

Comment:
This is currently causing the folowing bug:

![image](https://github.com/d-fine/Dataland/assets/99255296/c91dd25b-8760-490d-b202-6ed75be893e5)

Reply:
(and if you click submit, none of it seems to be actually uploaded)

Reply:
There should also be a component test avoiding this in the future.

Reply:
Yes, I noticed that too. Fix in my last push.

Reply:
... and I'm currently clicking through other frameworks to make sure they work, too

Reply:
@pioSko2 can you then also take care of emanuels other coment to change "reference" to "report" to better reflect our definition of document

Reply:
I don't see the fix.

My suggestion is to fix this by setting `:multiple="true"` to "false" in the FIleUploader, if the prop `moreThanOneDocumentAllowed` is set to "false"

Leaving this open until it is fixed and a test has been written.

Reply:
Have limited the selection to one file for certificates (base on moreThanOneDocumentAllowed) and starting to work on the test now

Reply:
Please tell me as soon as this has been done.  Then I will review the test and check if the feature works.